lmdb-sys/bindgen.rs

```diff
@@ -52,6 +52,7 @@ pub fn generate() {
         .whitelist_function("^(MDB|mdb)_.*")
         .ctypes_prefix("::libc")
         .blacklist_item("mode_t")
+        .blacklist_item("filehandle_t")
```

Comment:
Do you talk about that? If you want me to blacklist `filehandle_t` I think that is what I have done here.
Do I miss something?

Reply:
Yes, this is perfect, thanks!